src/test_data/mod.rs

```diff
@@ -1,3 +1,10 @@
+pub mod data;
+pub use data::*;
```

Comment:
If we renamed `test_data` to `test` we wouldn't need the `pub use`, I don't think.

```suggestion
pub mod fixtures;
```

Reply:
I think we still need `pub use` or at least `use` as we're accessing the module.